Packs/ExportIndicators/Integrations/ExportIndicators/ExportIndicators.py

```diff
@@ -247,32 +274,36 @@ def route_list_values() -> Response:
     """
     Main handler for values saved in the integration context
     """
-    params = demisto.params()
+    try:
+        params = demisto.params()
+
+        credentials = params.get('credentials') if params.get('credentials') else {}
+        username: str = credentials.get('identifier', '')
+        password: str = credentials.get('password', '')
+        if username and password:
+            headers: dict = cast(Dict[Any, Any], request.headers)
+            if not validate_basic_authentication(headers, username, password):
+                err_msg: str = 'Basic authentication failed. Make sure you are using the right credentials.'
+                demisto.debug(err_msg)
+                return Response(err_msg, status=401)
+
+        limit, offset, out_format, query = get_request_args(params)
+
+        values = get_outbound_ioc_values(
+            out_format=out_format,
+            on_demand=params.get('on_demand'),
+            limit=limit,
+            offset=offset,
+            last_update_data=demisto.getIntegrationContext(),
+            indicator_query=query,
+            cache_refresh_rate=params.get('cache_refresh_rate')
+        )
+
+        mimetype = get_outbound_mimetype()
+        return Response(values, status=200, mimetype=mimetype)
 
-    credentials = params.get('credentials') if params.get('credentials') else {}
-    username: str = credentials.get('identifier', '')
-    password: str = credentials.get('password', '')
-    if username and password:
-        headers: dict = cast(Dict[Any, Any], request.headers)
-        if not validate_basic_authentication(headers, username, password):
-            err_msg: str = 'Basic authentication failed. Make sure you are using the right credentials.'
-            demisto.debug(err_msg)
-            return Response(err_msg, status=401)
-
-    limit, offset, out_format, query = get_request_args(params)
-
-    values = get_outbound_ioc_values(
-        out_format=out_format,
-        on_demand=params.get('on_demand'),
-        limit=limit,
-        offset=offset,
-        last_update_data=demisto.getLastRun(),
-        indicator_query=query,
-        cache_refresh_rate=params.get('cache_refresh_rate')
-    )
-
-    mimetype = get_outbound_mimetype()
-    return Response(values, status=200, mimetype=mimetype)
+    except Exception as e:
+        return Response(str(e), status=500, mimetype='text/plain')
```

Comment:
```suggestion
        return Response(str(e), status=400, mimetype='text/plain')
```
What do you think?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/400